src/page/form/Form.js

```diff
@@ -0,0 +1,208 @@
+import React, { Component } from 'react'
+
+const subject = [
+  "Đá bóng",
+  "Cầu lông",
+  "Đua thuyền"
+];
+
+export default class Form extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      txtname:"",
+      txtemail:"",
+      txtpass:"",
+      txtavatar:"",
+      txtphone:"",
+      txtage:1,
+      txtgender:"male",
+      txtnote:"Buổi học đầu tiên về HTML",
+      txthobbi: new Set()
```

Comment:
Sai chính tả
Số ít: hobby
Số nhiều: hobbies
```suggestion
      hobbies: new Set()
```